aiormq/base.py

```diff
@@ -35,6 +35,10 @@ def remover(*_: Any) -> None:
             if future in self.futures:
                 self.futures.remove(future)
 
+            exc = future.exception()
```

Comment:
Is `asyncio.CancelledError` exception not handled here by intention?

Reply:
No that was not by intention (I was just not aware this needed any special handling). I tried adding a workaround there now.